repo.go

```diff
@@ -1591,3 +1591,8 @@ func (r *Repo) CheckRoleUnexpired(role string, validAt time.Time) error {
 	}
 	return nil
 }
+
+// GetMeta returns the underlying meta file map from the store.
+func (r *Repo) GetMeta() (map[string]json.RawMessage, error) {
```

Comment:
What happened to the `Repo.RootKeys()` function? Is it not there now? Why do we need this?

Reply:
`RootKeys()` is still there, but it returns `data.PublicKey`. When we give the root metadata to the client, it's of type `[]byte`. We *could* format the `data.PublicKey` into a root metadata file ourselves, then serialize. However, we'd have to pull out details like `consistent_snapshot`, thresholds, etc. very manually. 

To my mind, it seems to make sense for the repo to expose an interface to get the *exact* bytes in the `root.json` file, since that's what clients will be downloading, without having to drop down to the store itself.